registry/server/apps/routes/getApp.ts

```diff
@@ -28,9 +28,13 @@ const getApp = async (req: Request<GetAppRequestParams>, res: Response): Promise
 
     const [app] = await db.select().from<App>('apps').where('name', appName);
 
+    console.log('app = ');
+    console.log(app);
```

Comment:
can be removed

Reply:
looks like fixed locally but w/o push :(

Reply:
solved